org/open_power/Witherspoon/Fault.errors.yaml

```diff
@@ -24,6 +24,120 @@
 - name: PowerOnFailure
   description: System power failed to turn on
 
+- name: PSUPresentError
+  description: PSU failed to detect
```

Comment:
Does mihawk use policyTable.json?  If so, you'll want to add these errors to it.

Reply:
Thanks.